optuna/storages/base.py

```diff
@@ -645,6 +643,19 @@ def get_trial_system_attrs(self, trial_id: int) -> Dict[str, Any]:
         """
         return self.get_trial(trial_id).system_attrs
 
+    def read_from_remote_storage(self, study_id: int) -> None:
```

Comment:
It's not a strong opinion but, personally, I prefer `read_trials_from_remote_storage` to the current name because it sounds a bit more self-descriptive to me.
What do you think of it?

Reply:
Thank you for the comment. I think it's a better method-name and I'll adopt the suggestion.

Reply:
Thanks!